README.developer.md

```diff
@@ -153,14 +170,17 @@ After executing a `meson install` the MAN pages are located in the `man/man*` di
 
 ## Packaging
 
-Hirte is packaged as an RPM. The scripts used to create the SRPM and RPM may be found in [.build-scripts](./build-scripts).
+Hirte is packaged as an RPM. For a quick RPM creation, use `make rpm` target. This target will call all required
+scripts to generate RPMs and SRPM.
```

Comment:
I'm not sure about promoting make here, we should really aim for meson as the main build tool and that's why we have those build scripts, which are easy to use.

Reply:
I prepared a small PR to remove the meson dependency from the rpm build scripts:
https://github.com/containers/hirte/pull/307
This way, the make target points to the meson target and it should be fine to promote make here, I think.

Reply:
Personally I don't like advertising the usage of make in the project. We have agreed to use meson as the main build system, so we should stick to it. But I know others wants to have Makefile in the project to ease things for developers who are not that familiar with meson.

Anyway instead of `make rpm` I'd still leave the execution of `./build-scripts/build-rpm.sh` in the doc as the preferred solution ...

Reply:
it's not about know how to use meson, it's about keep both worlds for all devs as there is Makefile in the project. If the project want only use meson, just get out of Makefile would be appropriate IMO. I am going to update the patch and remove the `make rpm` thing for now.